python/HeaderService/hslib_salobj.py

```diff
@@ -456,11 +456,9 @@ async def complete_tasks_START(self, imageName):
             self.get_filenames(imageName)
 
             # Get the requested exposure time to estimate the total timeout
-            exptime_key = self.config.timeout_keyword
-            self.log.info("Collecting key to set timeout as key %s + %s" %
-                          (exptime_key, self.config.timeout_exptime))
-            metadata_tmp = self.collect([exptime_key])
-            timeout = metadata_tmp[exptime_key] + self.config.timeout_exptime
+            timeout_camera = self.read_timeout_from_camera()
+            timeout = timeout_camera + self.config.timeout_exptime
+            self.log.info(f"Setting timeout as camera_timeout + {self.config.timeout_exptime}")
```

Comment:
Be nice to report here what the camera timeout is actually being used.

Reply:
Thanks for the suggestion.